lib/core/src/lib/comments/comment-list/comment-list.component.ts

```diff
@@ -71,24 +49,34 @@ export class CommentListComponent implements OnInit, OnDestroy {
         this.clickRow.emit(this.selectedComment);
     }
 
-    getUserShortName(user: any): string {
-        let shortName = '';
+    getUserInitials(user: User): string {
+        let result = '';
         if (user) {
             if (user.firstName) {
-                shortName = user.firstName[0].toUpperCase();
+                result = user.firstName[0];
             }
             if (user.lastName) {
-                shortName += user.lastName[0].toUpperCase();
+                result += user.lastName[0];
             }
         }
-        return shortName;
+        return result.toUpperCase();
+    }
+
+    getUserFullName(user: User): string {
+        let result = '';
+
+        if (user) {
+            result = `${user.firstName || ''} ${user.lastName || ''}`;
+        }
+
+        return result.trim();
     }
 
-    isPictureDefined(user: any): boolean {
-        return user.pictureId || user.avatarId;
+    isPictureDefined(user: User): boolean {
+        return user['pictureId'] || user['avatarId'];
```

Comment:
hmm if we expect that User has these two fields - should not we update model instead of doing workarounds with []? It is a bit confusing that object has fields which are not defined in model

Reply:
in both cases the types are missing and it requires some better approach as this interface is shared between different teams with different expectations.